internal/k8s/converter.go

```diff
@@ -44,7 +45,12 @@ func NewUnstructuredConverter() *UnstructuredConverter {
 	projectcontour.AddKnownTypes(uc.scheme)
 	ingressroutev1.AddKnownTypes(uc.scheme)
 
-	return uc
+	// The kubebuilder tools' contract here is different, yay.
+	if err := serviceapis.AddToScheme(uc.scheme); err != nil {
```

Comment:
Should this be behind the experimental flag?

Reply:
Yes, it should, thanks.

Reply:
So, I've added this, with a TODO to remove it once the code is no longer experimental. Adding the scheme shouldn't do anything if the experimental flag is not true, as the informers will not be registered anyway. This is a belt-and-suspenders thing for now, just in case.